lib/order_checker.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'send_text'
+
+class OrderChecker
+
+  def initialize(menu, order_array, send_text = SendText.new)
+    @menu = menu
+    @order_array = order_array
+    @send_text = send_text
+  end
+
+  def check_order
+    check_if_valid
```

Comment:
This is fine, but better still is 'validate_selection'/'validate_total' or 'assert_selection_exists'/'assert_total_valid'. The difference is that 'check' doesn't tell me what the consequence of a failed check is, whereas by convention 'validate' and 'assert' generally indicate that an exception will be raised.